tests/test-cases/Group23-VIC-Machine-Service/23-05-VCH-Logs.md

```diff
@@ -0,0 +1,22 @@
+Test 23-05 - VCH Logs
+=======
+
+# Purpose:
+To verify vic-machine-server can provide logs for a VCH host when available
+
+# References:
+[1 - VIC Machine Service API Design Doc - VCH Certificate](../../../doc/design/vic-machine/service.md)
+
+# Environment:
+This test runs an external service binary that exposes the vic-machine API.
```

Comment:
The environment is what you need before you can run this test,
i.e. This test requires that a vSphere server is running and available